chain/kusama/defaults.go

```diff
@@ -99,3 +96,64 @@ const (
 	// Set to 0 to disable profiling.
 	DefaultPprofMutexRate = 0
 )
+
+// DefaultConfig returns a kusama node configuration
+func DefaultConfig() *cfg.Config {
```

Comment:
why do we have this function in here, but then have `dot.KusamaConfig()` as well? Should `dot.KusamaConfig()` call this function?

Reply:
Yeah, I don't know. I've refactored the config